godtools/Views/TractElements/TractElementFunctions.swift

```diff
@@ -84,4 +85,17 @@ extension BaseTractElement {
         return (elementName!, namespace, kind, properties, children)
     }
     
+    func elementListener() -> String {
+        return ""
+    }
+    
+    func sendMessageToElement(tag: String) {
```

Comment:
In the previous version of GodTools, we build the events and listeners and notifications that are posted and observed. Did you consider that here? I'd like to discuss that some.